libraries/npm-check-fork/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@rushstack/npm-check-fork",
+  "version": "1.0.0",
```

Comment:
```suggestion
  "version": "0.0.0",
```

Reply:
You'll also need to run `rush change`. Make this a `minor` bump with the changelog message "Initial release of fork."